### PR TITLE
[[ Bug 16397 ]] Add release note

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

File: docs/notes/bugfix-16397.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Change in empty is among the items of `1,2,3,` behavior
+
+In previous versions of LiveCode `is among the items of` behaved
+incorrectly when a trailing delimiter was present. Trailing delimiters
+are generally ignored in the LiveCode engine and legacy behaviors where
+they are not are documented anomalies that are maintained for backwards
+compatibility reasons. This change resolves the anomaly
+[bug report](http://quality.livecode.com/show_bug.cgi?id=16397)
```
Comment (livecodeali) [inline docs/notes/bugfix-16397.md]: Can you put the bug report here in brackets or something? Otherwise it sounds like the bug report is anomalous.

Comment (montegoulding) [reply docs/notes/bugfix-16397.md]: @livecodeali how about adding `detailed in`?

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 9bfe1e5

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 9bfe1e53f9744dcb3bf572c8bf6e03c920f0a61c

**Comment by livecode-vulcan** [comment]
😎 test success 9bfe1e53f9744dcb3bf572c8bf6e03c920f0a61c
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/123)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/123)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/123)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/123)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/123)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/123)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/123)

**Pull request merged**